package.json

```diff
@@ -85,7 +86,7 @@
     "ts-node": "^10.0.0",
     "tsconfig-paths": "^3.10.1",
     "typechain": "^5.1.2",
-    "typescript": "^4.3.5"
+    "typescript": "^4.9.5"
```

Comment:
looks like the new ts vertsion broke linting

Reply:
Fixed